helper/memory_calculator.go

```diff
@@ -103,7 +106,22 @@ func (m MemoryCalculator) Execute() (map[string]string, error) {
 	}
 
 	if totalMemory == UnsetTotalMemory {
-		m.Logger.Info("WARNING: Container memory limit unset. Configuring JVM for 1G container.")
+		if b, err := ioutil.ReadFile(m.MemoryInfoPath); err != nil && !os.IsNotExist(err) {
+			m.Logger.Info("Unable to read %s\n%w", m.MemoryInfoPath, err)
+		} else if err == nil {
+			rp := regexp.MustCompile("MemAvailable:\\s*(\\d*)\\s")
+			s := rp.FindStringSubmatch(string(b))[1]
```

Comment:
This should be more defensive and check the length of `rp.FindStringSubmatch(string(b))` so that this code doesn't panic if the regexp doesn't match for whatever reason.